script/VerifyMaxVault.s.sol

```diff
@@ -96,14 +109,35 @@ contract VerifyMaxVault is BaseScript, Test {
         RolesVerification.verifyTemporaryRoles(vault, deployer);
         RolesVerification.verifyTemporaryRoles(withdrawer, deployer);
 
+        // FIXME: TODO: reenable this once viewer is deployed
         // verify viewer
-        VaultVerification.verifyViewer(viewer, vault);
-        assertTrue(
-            MaxVaultViewer(address(viewer)).isUnderlyingAsset(contracts.WETH()), "WETH should be an underlying asset"
-        );
+        // VaultVerification.verifyViewer(viewer, vault);
```

Comment:
this needs to be reenabled once we deploy viewer which will be done after upgrade.